paddle/fluid/operators/conv_op.cc

```diff
@@ -60,8 +60,9 @@ void ConvOp::InferShape(framework::InferShapeContext* ctx) const {
                    "Due to the settings of paddings, filter_dims and "
                    "dilations, the output size is less than 0, please check "
                    "again.");
-    output_shape.push_back(OutputSize(in_dims[i + 2], filter_dims[i + 2],
-                                      dilations[i], paddings[i], strides[i]));
+    output_shape.push_back(ConvOutputSize(in_dims[i + 2], filter_dims[i + 2],
```

Comment:
When I was debugging conv_op.cc, looks like OutputSize has been linked to another function...